changes/311.release

```diff
@@ -0,0 +1 @@
+Released 2.2.3.
```

Comment:
This file shouldn't be needed for a PR into `main`. The documentation isn't as clear as it could be.

Reply:
It was left over. Documentation (which I will tackle, you know me now :) ) was fine.